front_end/src/components/landing_page/search_widget/SearchWidget.js

```diff
@@ -147,6 +172,20 @@ class SearchWidget extends Component {
     } else {
       return (
         <div className={classes.root}>
+          <Alert
```

Comment:
If only the date is missing, it shows the date-error above the destination input field.